plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -297,6 +297,7 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule("list", "watch").Groups(policyGroup).Resources("poddisruptionbudgets").RuleOrDie(),
 				rbac.NewRule("list", "watch").Groups(autoscalingGroup).Resources("horizontalpodautoscalers").RuleOrDie(),
 				rbac.NewRule("list", "watch").Groups(certificatesGroup).Resources("certificatesigningrequests").RuleOrDie(),
+				rbac.NewRule("list", "watch").Groups(storageGroup).Resources("storageclasses").RuleOrDie(),
```

Comment:
Anything that converts to shared informers where the kube-controller-manager doesn't currently have permission to list/watch that resource will need to have a rule added like this.